sycl/include/CL/sycl/access/access.hpp

```diff
@@ -47,7 +47,28 @@ enum class address_space : int {
   local_space
 };
 
-}  // namespace access
+} // namespace access
+
+using access::target;
+using access_mode = access::mode;
+
+#if __cplusplus > 201402L
+
```

Comment:
Can we get rid of the `__cplusplus > 201402L` macro and have working version with C++11?
I mean only tags and `noinit` variable

Reply:
Done

Reply:
New constructors are still under C++17 guard, because they have no sense without CTAD guides, which are C++17 only feature